Ryujinx.HLE/HOS/Services/Hid/ResultCode.cs

```diff
@@ -7,6 +7,9 @@ enum ResultCode
 
         Success = 0,
 
-        InvalidNpadIdType = (710 << ErrorCodeShift) | ModuleId
+        InvalidNpadDeviceType = (122 << ErrorCodeShift) | ModuleId,
+        InvalidNpadIdType     = (123 << ErrorCodeShift) | ModuleId,
+        InvalidDeviceIndex    = (124 << ErrorCodeShift) | ModuleId,
+        InvalidBufferSize     = (124 << ErrorCodeShift) | ModuleId
```

Comment:
```suggestion
        InvalidBufferSize     = (131 << ErrorCodeShift) | ModuleId
```
And it should be returned in `SendVibrationValues` call if `vibrationValues.Length != deviceHandles.Length`

Reply:
Thx for the right value !